From 4e529c1110601d9ecc941e74e6711f3aefa36f88 Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Tue, 26 Jun 2018 20:41:38 +0200
Subject: [PATCH 1/3] Revert "timers: Forward timer base before migrating
 timers"

Prepare to apply rt patch and reapply afterwards.

This reverts commit 13e75c74cd69ca460778fad5ab902f0b20869267.

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 kernel/time/timer.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/kernel/time/timer.c b/kernel/time/timer.c
index 7c477912f36d..2d5cc7dfee14 100644
--- a/kernel/time/timer.c
+++ b/kernel/time/timer.c
@@ -1884,12 +1884,6 @@ int timers_dead_cpu(unsigned int cpu)
 		spin_lock_irq(&new_base->lock);
 		spin_lock_nested(&old_base->lock, SINGLE_DEPTH_NESTING);
 
-		/*
-		 * The current CPUs base clock might be stale. Update it
-		 * before moving the timers over.
-		 */
-		forward_timer_base(new_base);
-
 		BUG_ON(old_base->running_timer);
 
 		for (i = 0; i < WHEEL_SIZE; i++)
-- 
2.13.6

